Doc/library/http.server.rst

```diff
@@ -33,9 +33,16 @@ handler.  Code to create and run the server looks like this::
    :attr:`server_port`. The server is accessible by the handler, typically
    through the handler's :attr:`server` instance variable.
 
+.. class:: ThreadedHTTPServer(server_address, RequestHandlerClass)
 
-The :class:`HTTPServer` must be given a *RequestHandlerClass* on instantiation,
-of which this module provides three different variants:
+   This class is identical to HTTPServer but uses threads to handle
+   requests by using the :class:`~socketserver.ThreadingMixin`. This
+   is usefull to handle web browsers pre-opening sockets, on which
```

Comment:
usefull -> useful

Reply:
As it's a backport, should I simply fix it in a new PR?

Reply:
Since the master and 3.7 versions are already merged, that would be simpler, sure.